spec/system/candidate_interface/carry_over/candidate_carries_over_application_with_references_to_new_cycle_and_referee_provides_reference_spec.rb

```diff
@@ -5,7 +5,9 @@
   include CycleTimetableHelper
 
   around do |example|
-    Timecop.freeze(mid_cycle) do
+    # It's important in this spec to create records in the 2021/22 recruitment cycle as after this
+    # the way we render references changes. See ApplicationForm#hide_new_reference_flow?
+    Timecop.freeze(mid_cycle(2021)) do
```

Comment:
perhaps replace the magic number and associated comment with an appropriately-named variable? eg `last_year_of_old_references_flow`

Reply:
or this could be a constant in `cycle_timetable`